### PR TITLE
XML: some newly changed descriptions and missed last time fixes

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: ROOT-RAMDISK/res/customconfig/customconfig.xml [hunk 1]
```diff
@@ -311,28 +311,28 @@
...
 
-		<spinner description="Switch from OLD LG Default Hot Plug driver to MSM Hot Plug, Alucard or Intelligent Hot Plug driver. Note that using Intelligent CPU Thermal Control with Default LG HotPlug is NOT recommended (causing low performance)." name="Hot Plug Driver" action="hotplug">
+		<spinner description="Switch from OLD LG Default HotPlug driver to other custom HotPlug drivers. Note that using Intelligent CPU Thermal Control with Default LG HotPlug is NOT recommended (causing low performance)." name="Hot Plug Driver" action="hotplug">
 			<spinnerItem name="Default LG" value="default"/> 
-			<spinnerItem name="MSM Hot Plug" value="msm_hotplug"/> 
-			<spinnerItem name="AutoSMP Hot Plug" value="autosmp"/> 
-			<spinnerItem name="Alucard Hot Plug" value="alucard"/> 
-			<spinnerItem name="Thunder Hot Plug" value="thunderplug"/> 
-			<spinnerItem name="Intelligent Hot Plug" value="intelli"/> 
+			<spinnerItem name="MSM HotPlug" value="msm_HotPlug"/>
```
Comment (dorimanx) [inline ROOT-RAMDISK/res/customconfig/customconfig.xml:326]: value="msm_HotPlug"/>  == WRONG! XML CRASH, again i need to search why.
dont use search and replace or copy paste. this is not TEXT, this is critical config.

same for action= never change that...

**Comment by hermitblue** [comment]
Now I see that value place was changed while renaming. Line 326 should have value="msm_hotplug"/>, not value="msm_HotPlug"/>

**Comment by hermitblue** [comment]
Now I see that value place was changed while renaming. Line 326 should have value="msm_hotplug"/>, not value="msm_HotPlug"/>

**Comment by ghost** [comment]
Now I see that value place was changed while renaming. Line 326 should have value="msm_hotplug"/>, not value="msm_HotPlug"/>
Used npp to rename all words and then rolled back when changes shouldn't be. One I missed. Sorry,
I know that all that I can make changes in name and label places.
If had know how to build kernel from github, i would firstly check myself if all is working and then publish it, but I can't.

**Pull request closed without merging**